test/iconGenerator.test.ts

```diff
@@ -2,34 +2,26 @@
 /* tslint:disable only-arrow-functions */
```

Comment:
I tested this locally and I haven't had any issues converting to arrow functions. `Mocha` worked just fine.Maybe it's worth re-evaluating it?